kconfigs/uboot_imx8.conf

```diff
@@ -0,0 +1,17 @@
+CONFIG_CMD_USB=y
```

Comment:
I came up with this list with a bit of trail and error. Not 100% sure everything still is needed. But it's a working combination at least.

Reply:
You saved me some work because this is the first time I've looked at what's in the file. :-)